src/components/SubSection/subsection.module.css

```
@@ -0,0 +1,20 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+    text-align: center;
+}
+
+.title {
+    color: var(--blue-700);
+    font-size: 36px;
+    letter-spacing: 0.125rem;
+}
+
+.subtitle {
+    font-size: 20px;
```

Esse também poderia ser menorzinho

```suggestion
    font-size: 18px;
```